integ/integ.sh

```diff
@@ -32,7 +33,10 @@ test_kinesis() {
 	docker-compose --file ./integ/test_kinesis/docker-compose.test.yml build
 	docker-compose --file ./integ/test_kinesis/docker-compose.test.yml up --abort-on-container-exit
 
+    # Giving a pause before running the validation test
+    # Firehose delivery stream has a buffer time which causes the delay to send the data
```

Comment:
Kinesis stream

Reply:
I believe the bffer time and buffer memory are for Kinesis Firehose not for the Kinesis Data Stream.